background.ts

```diff
@@ -112,13 +119,19 @@ Electron.app.whenReady().then(async() => {
     }
     callback(result);
   });
-  window.openPreferences();
+  window.openPreferences(false);
```

Comment:
I don't understand — this is application startup, and this is the first time the preferences window is being opened.  It will never be minimized (because the user hasn't had a chance to interact with it yet), so the argument has no effect, right?

Reply:
Correct, and obsolete code...